apps/api/src/app/widgets/e2e/initialize-widget-session.e2e.ts

```diff
@@ -41,4 +44,62 @@ describe('Initialize Session - /widgets/session/initialize (POST)', async () =>
 
     expect(body.message).to.contain('Please provide a valid app identifier');
   });
+
+  it('should pass the test with valid HMAC hash', async function () {
+    const subscriberId = '12345';
+    const secretKey = session.environment.apiKeys[0].key;
+
+    await enableWidgetSecurityEncryption(environmentRepository, session);
+
+    const hmacHash = createHmac('sha256', secretKey).update(subscriberId).digest('hex');
+    const response = await initWidgetSession(subscriberId, session, hmacHash);
+
+    expect(response.status).to.equal(201);
+  });
+
+  it('should fail the test with valid subscriber id or secret key hash', async function () {
+    try {
```

Comment:
I think this one is by mistake here

Reply:
There was a typo in the name. I will update it :)